runtime/bcutil/ClassFileOracle.cpp

```diff
@@ -2175,6 +2232,22 @@ ClassFileOracle::methodIsObjectConstructor(U_16 methodIndex)
 	return false;
 }
 
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+bool
+ClassFileOracle::methodIsConstructor(U_16 methodIndex)
```

Comment:
should probably add a comment indicating that these checks only apply to non-VT constructors, as VT constructors are static

Reply:
Comment added,